module/core/src/Infrastructure/Service/CurlDownloader.php

```diff
@@ -35,8 +35,11 @@ public function __construct(LoggerInterface $logger)
      *
      * @throws DownloaderException
      */
-    public function download(string $url, array $headers = []): string
+    public function download(string $url, array $headers = [], array $acceptedContentTypes = null): string
     {
+        if (null !== $acceptedContentTypes) {
```

Comment:
IS needed to throw exception ?? 
If this class not use the simple don use it ??